src/components/App.tsx

```diff
@@ -1,7 +1,6 @@
-import React from 'react';
-import { IntlProvider } from 'react-intl';
+import React, { useEffect } from 'react';
+import i18n from '../locales/i18n';
```

Comment:
landing to http://localhost:3000/fi/berths seems to produce error to console:
<img width="782" alt="Screenshot 2020-08-03 at 12 52 36" src="https://user-images.githubusercontent.com/1407958/89170920-0b987c80-d589-11ea-8f61-b9404d9414a2.png">